NEL.MESH/Services/Orchestrations/Mesh/MeshOrchestrationService.cs

```diff
@@ -75,6 +75,13 @@ public MeshOrchestrationService(ITokenService tokenService, IMeshService meshSer
             throw new System.NotImplementedException();
 
         public ValueTask<bool> AcknowledgeMessageAsync(string messageId) =>
-            throw new System.NotImplementedException();
+            TryCatch(async () =>
+            {
+                ValidateTrackMessageArgs(messageId);
+                string token = await this.tokenService.GenerateTokenAsync();
+                ValidateToken(token);
+                bool x = await this.meshService.AcknowledgeMessageAsync(messageId, authorizationToken: token);
```

Comment:
What is x?

also, always new line before a return

Reply:
Soz, was debugging